File: baselibrary/src/main/java/com/zqb/baselibrary/request/bean/BaseBean.java
package com.zqb.baselibrary.request.bean;

/**
 * Created by Allen on 2017/10/23.
 *
 * @author Allen
 *         <p>
 *         返回数据基类
 */

public class BaseBean {
    /**
     * 错误码
     */
    private int code;
    /**
     * 错误描述
     */
    private String msg;


    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public String getMsg() {
        return msg;
    }

    public void setMsg(String msg) {
        this.msg = msg;
    }

}

File: baselibrary/src/main/java/com/zqb/baselibrary/http/base/BaseBean.kt
package com.zqb.baselibrary.http.base

/**
 *  GSON 返回数据bean 基类
 *
 *  处理返回错误码 和错误描述（根据后端而定）
 */
open class BaseBean {
    /**
     * 错误码
     */
    var code: Int = 0
    /**
     * 错误描述
     */
    var message: String? = null

}